roles/confluent.test/molecule/kafka-connect-replicator-mtls-scram-rhel/molecule.yml

```diff
@@ -0,0 +1,245 @@
+provisioner:
+  name: ansible
+  config_options:
+    defaults:
+      hash_behaviour: merge
+  playbooks:
+    converge: multi_rbac_converge.yml
+  inventory:
+    group_vars:
+      all:
+        scenario_name: kafka-connect-replicator-mtls-scram-rhel
+
+
+      mds:
+        ssl_enabled: true
+        ssl_custom_certs: true
+        ssl_mutual_auth_enabled: true
+        # Paths relative to all.yml
+        ssl_ca_cert_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/ca.crt"
+        ssl_signed_cert_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
+        ssl_key_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-key.pem"
+        ssl_key_password: keypass
+
+
+      cluster2:
+
+        sasl_protocol: scram
+        ssl_enabled: true
+        ssl_custom_certs: true
+        # Paths relative to all.yml
+        ssl_ca_cert_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/ca.crt"
+        ssl_signed_cert_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
+        ssl_key_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-key.pem"
+        ssl_key_password: keypass
+
+
+      kafka_connect_replicator:
+
+        kafka_connect_replicator_listener:
+          ssl_enabled: true
+          ssl_mutual_auth_enabled: false
+          sasl_protocol: scram
+
+        kafka_connect_replicator_white_list: test-replicator-source
+        kafka_connect_replicator_bootstrap_servers: kafka-broker1:9092
+        kafka_connect_replicator_ssl_ca_cert_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/ca.crt"
+        kafka_connect_replicator_ssl_cert_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
+        kafka_connect_replicator_ssl_key_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-key.pem"
+        kafka_connect_replicator_ssl_key_password: keypass
+
+        kafka_connect_replicator_consumer_listener:
+          ssl_enabled: true
+          ssl_mutual_auth_enabled: true
+          sasl_protocol: none
+
+        kafka_connect_replicator_consumer_bootstrap_servers: mds-kafka-broker1:9092
+        kafka_connect_replicator_consumer_ssl_ca_cert_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/ca.crt"
+        kafka_connect_replicator_consumer_ssl_cert_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
+        kafka_connect_replicator_consumer_ssl_key_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-key.pem"
+        kafka_connect_replicator_consumer_ssl_key_password: keypass
```

Comment:
```suggestion
```
Since the CA is the same for all these kafkas, you can test that the defaulting logic works:
```
kafka_connect_replicator_consumer_ssl_ca_cert_path: "{{kafka_connect_replicator_ssl_ca_cert_path}}"
```